go/pools/numbered_test.go

```diff
@@ -55,61 +54,17 @@ func TestNumberedGeneral(t *testing.T) {
 		t.Errorf("want prefix 'ended at' and suffix '(test)', got '%v'", err)
 	}
 
-	id = 0
-	p.Register(id, id, true)
-	id = 1
-	p.Register(id, id, true)
-	id = 2
-	p.Register(id, id, false)
-	time.Sleep(300 * time.Millisecond)
-	id = 3
-	p.Register(id, id, true)
-	time.Sleep(100 * time.Millisecond)
-
-	// p has 0, 1, 2, 3 (0, 1, 2 are aged, but 2 is not enforced)
-	vals := p.GetOutdated(200*time.Millisecond, "by outdated")
-	if num := len(vals); num != 2 {
-		t.Errorf("want 2, got %v", num)
+	if p.Size() != 0 {
+		t.Errorf("want 0, got %v", p.Size())
 	}
-	if _, err = p.Get(vals[0].(int64), "test1"); err.Error() != "in use: by outdated" {
-		t.Errorf("want 'in use: by outdated', got '%v'", err)
-	}
-	for _, v := range vals {
-		p.Put(v.(int64), true)
-	}
-	p.Put(2, true) // put to 2 to ensure it's not idle
-	time.Sleep(100 * time.Millisecond)
-
-	// p has 0, 1, 2 (2 is idle)
-	vals = p.GetIdle(200*time.Millisecond, "by idle")
-	if len(vals) != 1 {
-		t.Errorf("want 1, got %v", len(vals))
-	}
-	if _, err = p.Get(vals[0].(int64), "test1"); err.Error() != "in use: by idle" {
-		t.Errorf("want 'in use: by idle', got '%v'", err)
-	}
-	if vals[0].(int64) != 3 {
-		t.Errorf("want 3, got %v", vals[0])
-	}
-	p.Unregister(vals[0].(int64), "test")
-
-	// p has 0, 1, and 2
-	if p.Size() != 3 {
-		t.Errorf("want 3, got %v", p.Size())
-	}
-	go func() {
-		p.Unregister(0, "test")
-		p.Unregister(1, "test")
-		p.Unregister(2, "test")
```

Comment:
are these test cases moved to different place?

Reply:
They are just deleted. It looked to me like these cases were related to the idle/outdated code in `pools.Numbered`. I deleted that code, so I didn't think these cases were needed. I thought the other remaining tests for `pools.Numbered` seemed good enough to cover the remaining functionality.